HST/product_labels/hst_dictionary_support.py

```
@@ -382,10 +382,17 @@ def detector_number_vs_extver(hdulist, fitsname):
                 try:
                     detector_ids = list(ref_hdulist[1].data['SEGMENT'])
+                    detector_ids.sort()
                 except KeyError:
-                    logger.error('COS/FUV table does not have a column "SEGMENT"',
-                                 filepath)
-                    detector_ids = ['UNK']
+                    if 'SEGMENT' in ref_hdulist[0].header:
+                        detector_ids = [ref_hdulist[0].header['SEGMENT']]
+                        if detector_ids == ['BOTH']:
+                            detector_ids = ['FUVA', 'FUVB']
+                        dectector_ids = [ref_hdulist[0].header['SEGMENT']]
```

Flake8 (7.3.0)

[error] local variable 'dectector_ids' is assigned to but never used

(F841)